app/utils/route-info-metadata.ts

```diff
@@ -4,10 +4,20 @@ export enum RouteColorScheme {
   Both,
 }
 
+export enum HelpscoutBeaconVisibility {
+  Show,
```

Comment:
```suggestion
  Visible,
  Hidden
```

(The values corresponding to "Visibility" shouldn't be verbs. If the key was something like `action`, it'd be fine for the values to be `show`/`hide`)